src/config.json

```diff
@@ -102,6 +102,30 @@
             "links": [
                 "https://www.presidency.ro/ro/media/declaratii-de-presa/declaratia-de-presa-sustinuta-de-presedintele-romaniei-domnul-klaus-iohannis1584201636"
             ]
+        },
+        {
+            "title": "Date la zi",
+            "aliases": [
+                "pacienți",
+                "paciente",
+                "pacienților",
+                "pacientelor",
+                "bolnavi",
+                "bolnave",
+                "pacient",
+                "pacientul",
+                "pacienta",
+                "îmbolnăvire",
```

Comment:
Let's remove this term, as per @overeha's request. Even if it only activates in combination with `covic-19` aliases, it's still very generic.

Reply:
am rezolvat @catileptic